### ISSUE TITLE
[XAML Controls Gallery(debug)->Media->AnimatedVisualPlayer] : Name of the button is not announced by narrator for Play button( 'Playback of a lottie animation').

### ISSUE THREAD

**Issue by v-amnat** [microsoft/Xaml-Controls-Gallery#679]
**Tags**
Accessibility, A11yCT, A11yMAS, MAS4.2.1, A11yLowImpact, ScreenReader,
A11ySev3, XAMLControlsGallery, WinUI, Win32, Wipro

**Test Environment**
XAML Controls Gallery(debug) version - 2.0.21.0
Win 32/ OS build- 21343.rs_Prerelease.210320-1757

**Steps to reproduce the bug**
Launch the XAML Controls Gallery (Debug) App
Go to the Media-> AnimatedVisualPlayer.
Hit on Tab key to traverse the "AnimatedVisualPlayer(Carousals)".
Navigate to play button under 'Playback of a lottie animation' and invoke it.
Listen the narrator announcement.

**Actual behavior**
When navigated to play button narrator is announcing simply as Button instead of announcing as Play button. After button is invoked then narrator is announcing as play button as recorded in the attachment.

**Expected**
All components need an accessible name, ideally using semantic elements and attributes. In rare cases, aria-label or aria-labelled by may be needed to alter the name. Learn more by reading about the accessible name property.

**User Impact**
When interactive elements have an incorrect name, it is difficult for people using assistive technology (like screen readers) to understand the purpose of control and its relationship to other content. 

**MAS Reference**
MAS 4.2.1 – Object Information

**Attachments**

[Play button name property is not announced.zip](https://github.com/microsoft/Xaml-Controls-Gallery/files/6310265/Play.button.name.property.is.not.announced.zip)

**Comment by chigy** [comment]
@anawishnoff fixed this
